### ISSUE TITLE
[GroupSortInteraction] Cue Logic to handle individually disabled items

### ISSUE THREAD

**Issue by marlitas** [phetsims/scenery-phet#865]
This issue comes from https://github.com/phetsims/mean-share-and-balance/issues/337#issuecomment-2294433473

We understand that GroupSortInteraction does not currently support proper cue behavior for individually disabled balls. This is an enhancement that should eventually be added to GroupSort.

Assigning to @zepumph to begin with in case he has anything to add.

**Comment by zepumph** [comment]
Yeah. This is a challenge since the cue visibility logic exists in the model, but the general notion of item enabled is Node.enabled. This meant that over in the above issue, we decided to add individual enabled over in the view in https://github.com/phetsims/scenery-phet/commit/3690a615369b738298e6f81518bf1d42fe20ae02.

Some challenges with supporting this in the model:

1. ItemModel as a type would want to support some sort of enabledProperty.
2. Ideally we combine enabled strategies, and don't just keep the view option from the above commit (isGroupItemEnabled).
3. A good default for this logic is Node.enabled, but I'm unsure how that would be able to live in the model type.
4. This issue recognizes that the model/view pattern is a bit strange here, since all model Properties are really about the controller/view interaction, and thus create problems with wanting Node Properties in the GroupSortInteractionModel (which still seems "view-like" per say). 


I'm not planning to work on this, but it seems valuable to track for next time we work on group sort.
